backend/test/Test/UploadSpec.hs

```diff
@@ -121,6 +126,25 @@ spec = withContext $ startWithInitial $ do
         Library.getCompounds noSorting fullContent >>= shouldBeZeroLength
         Dashboard.getExperiments Nothing Nothing Nothing noSorting fullContent
           >>= shouldBeZeroLength . erExperiments
+
+    -- Note that there are some checks in LibrarySpec and DashboardSpec as well.
+    describe "builds correct mde-links for compounds" $ do
```

Comment:
It doesn't look like hspec eDSL usage intended by its authors. The string passed to `describe` is supposed to name the thing you are describing and then the string passed to `it` is supposed to contain the behavior you are describing. Like `describe "dog" $ it "loves food" $ do testLovesFood`.

So I think it's more appropriate to write `describe "mde link construction"` here or something like that.